pkg/curatedpackages/reader.go

```diff
@@ -0,0 +1,87 @@
+package curatedpackages
+
+import (
+	"context"
+	"fmt"
+
+	eksdv1 "github.com/aws/eks-distro-build-tooling/release/api/v1alpha1"
+	"sigs.k8s.io/yaml"
+
+	packagesv1 "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/logger"
+	"github.com/aws/eks-anywhere/pkg/manifests"
+	releasev1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+type PackageReader struct {
+	ManifestReader *manifests.Reader
+}
+
+func NewPackageReader(mr *manifests.Reader) *PackageReader {
+	return &PackageReader{
+		ManifestReader: mr,
+	}
+}
+
+func (r *PackageReader) ReadBundlesForVersion(version string) (*releasev1.Bundles, error) {
```

Comment:
I would suggest defining PackageReader like

```go
type PackageReader struct {
	*manifests.Reader
}
```
That way, we don't need to define all these wrappers and `PackageReader` will get all those methods "for free"

In fact we should probably define an interface for the ManifestReader in this package, but we can leave that for when we add the unit tests.

Reply:
Discussed offline, second part will be taken care of in followup pr.